web-api/openapi/schemas/document.yml

```diff
@@ -19,12 +19,18 @@ DocumentPropertyString:
   maxLength: 2048
 
 DocumentProperty:
-  description: Arbitrary property that can be attached to a document.
+  description: | 
+    Arbitrary property that can be attached to a document.
+    When a property is set to `null`, it is treated as if that property had no values.
```

Comment:
:wrench: A property existing and having no value and it not existing for a specific document isn't guaranteed to be the same, so we should change the formulation to something like `as if the document doesn't have that property` or similar. Especially wrt. to filters this could matter.

Reply:
It is the same exact formulation the ES uses.